test/helper.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+/// MUST BE FIRST! set the NODE_ENV to test to disable logging, switch to test db
+process.env.NODE_ENV = 'test';
+
+const fixtures = require('sequelize-fixtures');
+const path = require('path');
+
+const models = require('../models');
+
+const loadFixtures = async function(files) {
```

Comment:
Here's a little helper I wrote to make it easier to load fixtures from the fixtures folder.